redisdb/tox.ini

```diff
@@ -0,0 +1,42 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist = unit, redis{32,40}, flake8
+
+[common]
+deps =
+    ../datadog-checks-base
+    -rrequirements-dev.txt
+commands =
+    pip install --require-hashes -r requirements.txt
+    pytest -m"integration" -v -s
+
+[testenv]
+platform = linux2|darwin
+
+[testenv:unit]
+deps = {[common]deps}
+commands =
+    pip install --require-hashes -r requirements.txt
+    pytest -m"not integration" -v
+
+[testenv:redis32]
+deps = {[common]deps}
+setenv = REDIS_VERSION=3.2
+passenv = DOCKER_HOST*
+commands = {[common]commands}
+
+[testenv:redis40]
+deps = {[common]deps}
+setenv = REDIS_VERSION=4.0
+passenv = DOCKER_HOST*
+commands = {[common]commands}
+
+[testenv:flake8]
+skip_install = true
+deps = flake8
+commands = flake8 .
+
+[flake8]
+exclude = .eggs,.tox,test
```

Comment:
`tests`

Reply:
Actually I think I'm going to let flake8 lint the tests modules too...

Reply:
the test modules should definitely be linted